lib/galaxy/tools/parameters/validation.py

```diff
@@ -18,7 +18,7 @@
 def get_test_fname(fname):
     """Returns test data filename"""
     path, name = os.path.split(__file__)
-    full_path = os.path.join(path, 'test', fname)
+    full_path = os.path.join(path, '..', '..', '..', '..', 'test-data', fname)
```

Comment:
This breaks the package structure, and therefore the `test_galaxy_packages` tests.

Reply:
True. What would be the simplest solution for this? Create a test data directory in `tools` and link the test data?

Reply:
There's already one, with symlinks: lib/galaxy/tools/parameters/test/